### DISCUSSION TITLE
FireFox Workflows?

### DISCUSSION THREAD

**Discussion by WirlyWirly** [violentmonkey/violentmonkey#2230] [General]
Hello,

Since FireFox doesn't support the `file://` option, I'm curious what peoples workflow is when working on a local `user.js` file in their editor of choice.

I run docker services, so thought I might try to find an http-server to host the file and use the `@require` line.

I think that'd work?

**Comment by tophf** [comment]
file:// works in Firefox for the main script, but `localhost` may be more convenient, see https://violentmonkey.github.io/posts/how-to-edit-scripts-with-your-favorite-editor/
